### DISCUSSION TITLE
请问如何拿到全局的AliasToken

### DISCUSSION THREAD

**Discussion by larrygogo** [ant-design/ant-design#39075] [Q&A] [answered]
版本：5.0.1
实践中的项目使用了 styled-components
希望将 antd 的 aliasToken 注入到 styled-components 的 theme 中

官网只给出静态获取 mapToken 的途径，请问有没有办法拿到 aliasToken

**Comment by arvinxx** [comment, marked as answer]
```ts
import { theme } from 'antd';

const { token } = theme.useToken()
```

这个 token 里包含了alias token

**Reply by dbsan2333** [reply]
可这个需要在vue生命周期，我想要在构建时就使用它以便于生成less变量，官方提供的defaultAlgorithm方法生成的不包含AliasToken
